docs/volunteering/glossary.md

```diff
@@ -0,0 +1,70 @@
+# Glossary
+
+While we try to use terminology which are clear and/or well known, here's a list
+you can check if you're not sure!
+
+<!---
+Note to editors; please:
+- keep this list in alphabetical order
+- ensure that, within the list, usages of other terms defined in the list are
+  emphasised suitably
+--->
+
+_Competition programme_:
+
+: the overall programme of events and activities which the teams participate in
+
+_Competition year_:
+
+: the 6 months (or so) between _Kickstart_ and the _Competition_ during which
+teams build their robots; see also _SRnnnn_
+
+_Core Team_:
```

Comment:
This is outdated. I suggest replacing this with definitions of the Competition and Kit Team.

Reply:
I don't think it should be replaced, but I think saying it's what the Competition Team was referred to as for the 2019 competition cycle is definitely worthwhile.

Reply:
At the very least, the definitions of the competition and kit team should be here.

Reply:
a13a57b